### PR TITLE
feat(Search) Global Search operators

### PR DESCRIPTION
**Pull request by maliknajjar**
image

### CODE REVIEW

**Comment by maliknajjar** [comment]
In addition to the features above. You can add exact matches searches inside OR combined searches.
example: "car" OR "bicycle"

**Comment by maliknajjar** [comment]
The first commit is a fix to the fieldtype search not working in global search.
example: phone::1231231231

File: include/utils/utils.php [hunk 1]
```diff
@@ -2017,6 +2017,18 @@ function escape_single_quotes($value) {
 	return $value;
 }
 
+/**
+ * Function to get substring between two charachters.
+ */
+function get_string_between($string, $start, $end){
+    $string = ' ' . $string;
+    $ini = strpos($string, $start);
+    if ($ini == 0) return '';
```
Comment (joebordes) [inline include/utils/utils.php:2026]: $ini === false

**Comment by joebordes** [comment]
LGTM

Thanks!

**Pull request merged**